### PR TITLE
Add regression Make target and gut Makefile

### PR DESCRIPTION
**Pull request by lindluni**
Signed-off-by: Brett Logan <brett.t.logan@ibm.com>

### CODE REVIEW

**Comment by lindluni** [comment]
Test run of the new daily suite with matrix strategy:

https://dev.azure.com/lindluni/Fabric-Test/_build/results?buildId=467&view=logs&j=1ce0692c-fe8e-5501-0d5f-d1e86afbdb63

File: Makefile [hunk 1]
```diff
@@ -4,54 +4,29 @@
...
-#   - javaenv                      - clone the fabric-chaincode-java repository and build the javaenv image.
-#   - nodeenv                      - clone the fabric-chaincode-node repository and build the nodeenv image.
-#   - svt-weekly-pte-12hr-test-k8s - Test 12hr longrun test in k8s environment.
-#   - npm-init                     - initialize the PTE NPM modules
-#   - git-latest                   - init git submodules to latest available commit.
-#   - git-init                     - init git submodules.
-#   - pre-setup                    - installs node and govendor
-#   - clean                        - cleans the docker containers.
-#   - gotools                      - installs go tools, such as: ginkgo, golint, goimports, gocov and govendor
+#   - regression/barebones			- Executes barebones tests
```
Comment (suryalnvs) [inline Makefile:7]: Alignment is missing for descriptions.

File: Makefile [hunk 2]
```diff
@@ -4,54 +4,29 @@
...
+	cd ${@} && ginkgo -v
 
-.PHONY: ci-barebones
-ci-barebones: pre-reqs pull-binaries-fabric barebones-tests
+.PHONY: pre-reqs
+pre-reqs: docker-clean npm-init pull-binaries-fabric gotools
 
-.PHONY: ci-daily
-ci-daily: pre-reqs pull-binaries-fabric daily-tests
+.PHONY: docker-clean
```
Comment (suryalnvs) [inline Makefile]: Where are we using docker-clean target? In test suites, we take care of fabric networks and chaincode containers in AfterSuite.

File: regression/barebones/go.sum [hunk 3]
```diff
@@ -4,26 +4,32 @@ github.com/golang/protobuf v1.2.0 h1:P3YflyNX/ehuJFLhxviNdFxQPkGK5cDcApsge1SqnvM
 github.com/golang/protobuf v1.2.0/go.mod h1:6lQm79b+lXiMfvg/cZm0SGofjICqVBUtrP5yJMmIC1U=
 github.com/hpcloud/tail v1.0.0 h1:nfCOvKYfkgYP8hkirhJocXT2+zOD8yUNjXaWfTlyFKI=
 github.com/hpcloud/tail v1.0.0/go.mod h1:ab1qPbhIpdTxEkNHXyeSf5vhxWSCs/tWer42PpOxQnU=
-github.com/hyperledger/fabric-test v1.1.1-0.20191126160459-3b39434023ee h1:Ba0qI2fqC5CGy/VDyxlBB7enyBlV3teyQddk5XhOK5g=
-github.com/hyperledger/fabric-test v1.1.1-0.20191126160459-3b39434023ee/go.mod h1:7K2xXfkAUybufYc/nYZEfJx/B84HnjHHsy6eKWZJyc4=
+github.com/hyperledger/fabric-test v1.4.4 h1:17Yu+p0QQZlswQFHYs9Jt/2hTgunsqInhMU8qXdJeDc=
```
Comment (suryalnvs) [inline regression/barebones/go.sum]: shouldn't this point to master?

File: regression/barebones/go.mod [hunk 4]
```diff
@@ -0,0 +1,10 @@
+module github.com/hyperledger/fabric-test/regression/barebones
+
+go 1.13
+
+require (
+	github.com/hyperledger/fabric-test v1.4.4
```
Comment (suryalnvs) [inline regression/barebones/go.mod]: shouldn't this point to fabric-test master?

Comment (lindluni) [reply regression/barebones/go.mod]: Corrected, I forgot to add the replace directive

File: regression/barebones/go.sum (hunk 3 above)
Comment (lindluni) [reply regression/barebones/go.sum]: Corrected, I forgot to add the `replace ` directive

File: Makefile (hunk 2 above)
Comment (lindluni) [reply Makefile]: Removed, I don't like it anyway, it might clean up containers I am running on my own

File: Makefile (hunk 1 above)
Comment (lindluni) [reply Makefile:7]: It's only in the github UI, its correct in the actual Makefile, I've been noticing this in a lot of places lately in GitHub

**Review by suryalnvs — APPROVED** [review]

**Pull request merged**